configs/community.dbuild

```diff
@@ -588,9 +588,12 @@ build += {
     extra.run-tests: false
   }
 
+  // disable -Xfatal-warnings because of new unused warnings introduced
+  // by https://github.com/scala/scala/pull/5402
   ${vars.base} {
     name: "pcplod"
     uri:  ${vars.uris.pcplod-uri}
+    extra.commands: ${vars.default-commands} [ "removeScalacOptions -Xfatal-warnings" ]
```

Comment:
is the error in pcplod going to be fixed by this? I think it's an error in its test suite, which is likely not affected by the compiler used to compile the project?